docs/src/routes/+page.svelte

```
@@ -1,365 +1,1531 @@
+                    <span>svelte</span><span class="slash">/</span><span>headless table</span><span
+                        class="end">.</span
+                    >
+                </h1>
+                <p class="sub">
+                    A <b>powerful, composable</b> table library for Svelte 5 — bring your own
+                    markup, bring your own styles, and snap on as many of the {PLUGIN_COUNT} plugins as
+                    you need. Sorting, filtering, pagination, grouping, expansion, selection, column resizing,
+                    sub-rows, virtual scroll. Fully typed, MIT, zero opinions about how the table renders.
+                </p>
+                <div class="cta-row">
+                    <a class="pri" href="/docs/getting-started/overview">get started ↗</a>
```

ESLint

[error] Unexpected href link without resolve().

(svelte/no-navigation-without-resolve)